sdk/src/wallet/account/operations/syncing/addresses/output_ids/mod.rs

```diff
@@ -136,6 +159,33 @@ where
                     .boxed(),
                 );
             }
+        } else if address.is_alias() && sync_options.alias.foundry_outputs {
```

Comment:
So this was simply forgotten before or ?

Reply:
No, before we always got the foundry outputs at the same time when getting alias outputs, but now we can get alias outputs without getting the foundry outputs directly, that's why it's needed now

Reply:
Yeah but they actually have a `RouteOutputs` variant 😆